tests/integration/upgrade/test_sharding_rollback.py

```diff
@@ -103,7 +104,7 @@ async def test_rollback_on_config_server(
     # TODO implement this check once we have implemented the post-cluster-upgrade code DPE-4143
 
 
-@pytest.mark.skip("re-enable these tests once upgrades are available on charmhub")
```

Comment:
the tests in this suite and the tests in `tests/integration/upgrade/test_sharding_upgrade.py` still do not use the charm from charmhub and are effectively not testing the upgrade.

Now that CI is functioning and we have published a new revision to charmhub, we should update these tests to use the charm from charmhub

Reply:
Since we rolledback to the same charm (and a charm without any changes in the workload) there was no rolling upgrade and so `resume-upgrade` fails in the test since there is no upgrade to resume. 

Please disable this test and make a separate GitHub Issue / JIRA issue and I can try to get to it this week